scipy/stats/_distribution_infrastructure.py

```diff
@@ -1344,19 +1392,22 @@ def _generate_example(dist_family):
         """
 
     example += f"""
-    To evaluate the probability density function of the underlying distribution
+    To evaluate the probability density/mass function of the underlying distribution
     at argument ``x={x}``:
 
     >>> x = {x}
-    >>> X.pdf(x)
-    {X.pdf(x)}
-
+    >>> X.pdf(x), X.pmf(x)
+    {X.pdf(x), X.pmf(x)}
+    
     The cumulative distribution function, its complement, and the logarithm
     of these functions are evaluated similarly.
 
     >>> np.allclose(np.exp(X.logccdf(x)), 1 - X.cdf(x))
     True
+    """
 
+    if issubclass(dist_family, ContinuousDistribution):
```

Comment:
The first example here fails for discrete distributions because the the inverse distribution functions are very sensitive to numerical error of the input (e.g. subtractive cancellation of `1 - X.ccdf(x)`) near the edges.

The second example is two-arg CDF, which is not implemented for discrete distributions right now.